examples/dem/rectangular_tank_2d.jl

```diff
@@ -35,6 +35,9 @@ tank = RectangularTank(particle_spacing, (rock_width, rock_height),
 
 # Move the rock particles up to let them fall
 tank.fluid.coordinates[2, :] .+= 0.5
+# small perturbation
+tank.fluid.coordinates .+= rand((-particle_spacing / 100):1e-4:(particle_spacing / 100),
+                                (2, nparticles(tank.fluid)))
```

Comment:
This doesn't work when the particle spacing is smaller than `1e-2`.
```suggestion
tank.fluid.coordinates .+= 0.01 .* (2 .* rand(length(tank.fluid.coordinates)) .- 1)
```